terraform/versions.tf

```
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 1.5"
```

For the same reason as above I'd recommend setting the constraint as low as possible unless you are actually relying on some features that were added in terraform 1.5.
```suggestion
  required_version = ">= 0.13"
```